physics/mynnedmf_wrapper.meta

```diff
@@ -1398,7 +1398,7 @@
   standard_name = dry_deposition_velocity_mynn_pbl_transport
   long_name = dry deposition velocity by mynn pbl transport
   units = m s-1
-  dimensions = (horizontal_loop_extent,2)
+  dimensions = (horizontal_loop_extent,3)
   type = real
```

Comment:
Is there any chance you can turn these hard-coded values into a derived variable?